README.md

```diff
@@ -1,17 +1,80 @@
+| Parameters            | Refers to parameters in an SDK method or API definition. Use in cases where we speak about needs, such as "takes parameters x, y, and z" or "requires parameters x and y".                                                                                         |
+| Arguments             | Refers to arguments passed to CLI, SDK method, or REST API. Use when the argument has a concrete value and when we refer to invoking a method, such as "pass in arguments x and y, or "call account.create() with the argument "unique()" to generate a random id. |
 
-#### Spell Checking
+#### Authentication
+| **Term**               | **Suggested Usage.**                                                                                                                                                                                                                                          |
+|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Authentication Service | The Authentication service encompases the Account API, Users API, and Teams API.                                                                                                                                                                              |
+| Account API            | The Account API is an API under the Authentication service that is used by Client SDKs. Avoid "Account service" to avoid confusing developers that it is different from the "Users API" because it might imply they are separate services.                    |
+| Users API              | The User API is an API under the Authentication service that is used by Server SDKs. Avoid "Users service" to avoid confusing developers that it is accessing users different from the "Account API" because it might imply they are separate services.       |
+| Teams API              | The Teams API is an API under the Authentication service that is used to organize created accounts. Avoid "Teams service", because it is not a stand alone service.                                                                                           |
+| Authentication Method  | An authentication method refers to ways to authenticate a user like email, phone, and JWT. Not a proper noun, avoid terms like auth method or login method.                                                                                                   |
+| OAuth2 Providers       | An OAuth2 provider refers to ways to authenticate users using accounts from service providers like AWS, Facebook, or Microsoft. Avoid using simplified terms like OAuth provider and provider to avoid confusion (there are multiple types of providers now). |
```

Comment:
Not sure if AWS example is relevant. Let's do something tech companies uses for oauth, like GitHub.

Reply:
I would actually put GitHub as first example. Then even Google maybe.